utils/zpe-updater/cmd/tools/main.go

```diff
@@ -87,7 +90,24 @@ func main() {
 		zpuConfig.Zts = ztsURL
 	}
 
-	// first check if we're just asked to view a local domain
+	// first, if running check we need to verify policy files
+	// validity and generate a json metrics that can be pushed to
+	// a monitoring service
+	if check {
+		policyStatus, err := zpu.CheckState(zpuConfig)
+		if err != nil {
+			log.Fatalf("Unable to check policy files state, %v", err)
+		}
+		policyMetrics := metrics.FormPolicyMetrics(policyStatus)
+		jsonMetrics, err := json.Marshal(policyMetrics)
```

Comment:
Similarly here. If any of the policy files is not updated as expected, the return code should be 1 instead of 0 with the json output. It would simplify where I only need to check the return code from the call to see if everything is ok or not.

Reply:
fixed